### PR TITLE
Fixed concurrency bug of pop_from_queue function and edited unittest.

### PR DESCRIPTION
**Pull request by SuhwanSong**
There was a concurrency bug between `get_vers()` and `pop_from_queue()` and this commit fixed it.
Now the bisecter can correctly bisect the regressions.

### CODE REVIEW

File: src/driver.py [hunk 1]
```diff
@@ -113,6 +113,9 @@ def setup_browser(self):
                 break
             except Exception as e:
                 print (e)
+                time.sleep(0.5)
```
Comment (progers) [inline src/driver.py:116]: Is this needed?

File: src/chrome_binary.py [hunk 2]
```diff
@@ -82,7 +82,13 @@ def download(url, name, base="."):
         revision = str(revision)
         binary_dir_path = os.path.join(path, revision)
         if os.path.exists(binary_dir_path):
-            return True
+            brp = self.get_browser_path(path, revision)
```
Comment (progers) [inline src/chrome_binary.py:85]: Is this needed? I thought this was moved into the other pull request?
(I might be reading the diff wrong)

File: src/modules.py [hunk 3]
```diff
@@ -693,6 +682,7 @@ def answer(self, answer_version) -> None:
             else:
                 nonbug_dict[culprit] += 1
 
+        print ('Tested in answer()', num)
```
Comment (progers) [inline src/modules.py:685]: Can you remove this logging?

File: src/driver.py (hunk 1 above)
Comment (SuhwanSong) [reply src/driver.py]: I thought it is useful not to make many threads open browsers at once, but I think it's not useful for that. I removed this.

**Review by progers — APPROVED** [review]
LGTM

File: src/chrome_binary.py (hunk 2 above)
Comment (SuhwanSong) [reply src/chrome_binary.py]: I forgot to remove it. I removed it.

File: src/modules.py (hunk 3 above)
Comment (SuhwanSong) [reply src/modules.py]: I removed it

**Pull request merged**